controllers/cloud.redhat.com/makers/maker.go

```diff
@@ -594,18 +605,28 @@ func (m *Maker) makeDependencies(c *config.AppConfig) error {
 
 	// Iterate over all deps
 
-	depConfig, missingDeps := makeDepConfig(m.Env.Spec.Providers.Web.Port, m.App, &apps)
+	if m.Env.Spec.Providers.Web.InternalPort == 0 {
+		m.Env.Spec.Providers.Web.InternalPort = 10000
+	}
+	depConfig, intDepConfig, missingDeps := makeDepConfig(m.Env.Spec.Providers.Web.Port, m.Env.Spec.Providers.Web.InternalPort, m.App, &apps)
```

Comment:
Should we rename `Web.Port` to `Web.PublicPort`?

Reply:
I'd like that - again it's just more backwards compat stuff

Reply:
Perhaps we should come back to this in a later PR to remove the backwards compat stuff?